lessons/08-html-site/contact.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+
+<html>
+  <head>
+    <title>Contact | Basharah Batool</title>
+  </head>
+
+  <body>
+    <header>
+      <img src="rose.jpg" alt="My pic" width="150" height="150" />
+      <h2>Basharah's Portfolio</h2>
+      <nav>
+        <a href="index.html">Home</a> | <a href="about.html">About</a> |
+        <a href="Contact.html">Contact</a>
+      </nav>
+      <hr />
+      <h1>Contact</h1>
+    </header>
+
+    <main>
+      <form action="contact-success.html" method="GET">
+        <div>
+          <label>Name:</label>
+          <input
+            type="text"
+            name="name"
+            placeholder="Enter your name here"
+            required
+          />
+        </div>
+        <div>
+          <label>Email:</label>
+          <input
+            type="email"
+            name="email"
+            placeholder="Enter valid email address"
+            required
+          />
+        </div>
+        <div>
+          <Label>Message:</Label>
```

Comment:
Careful with capital letters in tags, should be `<label>`

Reply:
Oh yess ....it's user error.... Resolved now :)